index.html

```diff
@@ -0,0 +1,399 @@
+            <article class="darkGray padding">
+                <h2>Design Patterns & Checklists</h2>
+                <ul>
+                    <li><a href="#">Responsive Art-Direction</a></li>
+                    <li><a href="#">Perfect Configurator</a></li>
+                    <li><a href="#">Perfect Web Form</a></li>
+                    <li><a href="#">Perfect Accordion</a></li>
+                    <li><a href="#">Perfect Date Picker</a></li>
+                    <li><a href="#">Perfect Slider</a></li>
+                    <li><a href="#">Perfect Comparison Table</a></li>
+                    <li><a href="#">Perfect Infinite Scroll</a></li>
+                    <li><a href="#">Perfect Carousel</a></li>
+                    <li><a href="#">Privacy UX</a></li>
+                </ul>
+            </article>
+            <article class="lightGray padding">
+                <h2>Improving Workflow & Data Protection</h2>
+                <ul>
+                    <li><a href="#">GDPR For Developers</a></li>
+                    <li><a href="#">Privacy Framework</a></li>
+                    <li><a href="#">Estimating Time</a></li>
+                    <li><a href="#">Dealing With Irrational Clients</a></li>
+                    <li><a href="#">Getting Deliverables Right</a></li>
+                    <li><a href="#">Meetig Deadlines</a></li>
+                    <li><a href="#">Convincing Clients</a></li>
+                    <li><a href="#">Invoicing Clients</a></li>
+                </ul>
+            </article>
+        </div>
+        <div class="article3 darkGray flex justify-center ">
+            <img class="trumpetImg" src="assets/images/cat-with-trampet.svg" alt="cat-with-trompet">
+        </div>
+        <article class="article3 lightGray flex justify-center">
+            <h3 class="article-title">SEE ALL ARTICLES </h3>
+        </article>
+    </main>
+    <footer>
+        <div class="article3 darkGray flex justify-center">
+            <h3 class="article-title">BROWSE ALL SMASHING MAGAZINE TOPICS</h3></div>
+            <div class="footer darkGray">
+             <ul>
+                <li><a href="#">Accesibility</a></li>
+                <li><a href="#">Android</a></li>
+                <li><a href="#">Animation</a></li>
+                <li><a href="#">CSS</a></li>
+                <li><a href="#">Design</a></li>
+                <li><a href="#">Design Patterns</a></li>
+                <li><a href="#">User experience</a></li>
+                <li><a href="#">Wallpapers</a></li>
+                <li><a href="#">Accesibility</a></li>
+                <li><a href="#">Android</a></li>
+                <li><a href="#">Animation</a></li>
+                <li><a href="#">CSS</a></li>
+                <li><a href="#">Design</a></li>
+                <li><a href="#">Design Patterns</a></li>
+                <li><a href="#">User experience</a></li>
+                <li><a href="#">Wallpapers</a></li> 
+            </ul>
+        </div>
```

Comment:
Your footer section looks like a sentence rather than list of items. Let's make it similar to this :point_down: 
![Screenshot from 2020-05-25 09-19-26](https://user-images.githubusercontent.com/37747619/82776073-6d41b980-9e69-11ea-9212-167986765304.png)